indexer/index.py

```diff
@@ -183,6 +184,10 @@ def _index(
     if not paths:
         return []
 
+    # Make FC Rest Client and IceProd Manager
+    fc_rc = fc_rc_creator()
+    manager = manager_creator()
```

Comment:
I'm sure there's a Java Enterprisey BeanStrategyFactoryFactoryImpl joke in here somewhere :smile: